ocs_ci/deployment/provider_client/storage_client_deployment.py

```diff
@@ -0,0 +1,630 @@
+"""
+This module provides installation of ODF in provider mode and storage-client creation
+on the hosting cluster.
+"""
+import atexit
+import logging
+import tempfile
+import time
+
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants, ocp, defaults
+from ocs_ci.ocs.rados_utils import RadosHelper
+from ocs_ci.deployment.helpers.lso_helpers import setup_local_storage
+from ocs_ci.ocs.node import label_nodes, get_all_nodes, get_node_objs
+
+# from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.ui.validation_ui import ValidationUI
+from ocs_ci.ocs.ui.base_ui import login_ui, close_browser
+from ocs_ci.ocs.utils import (
+    setup_ceph_toolbox,
+    enable_console_plugin,
+    run_cmd,
+)
+from ocs_ci.utility.utils import (
+    wait_for_machineconfigpool_status,
+)
+from ocs_ci.utility import templating, version
+from ocs_ci.deployment.deployment import Deployment, create_catalog_source
+from ocs_ci.deployment.baremetal import clean_disk
+from ocs_ci.ocs.resources.storage_cluster import (
+    verify_storage_cluster,
+    # check_storage_client_status,
```

Comment:
Remove commented imports when WIP label is removed.

Reply:
Addressed in latest commit.